core/domain/exp_jobs_one_off_test.py

```diff
@@ -2340,3 +2340,133 @@ def test_no_action_is_performed_for_deleted_exploration(self):
         run_job_for_deleted_exp(
             self,
             exp_jobs_one_off.InteractionCustomizationArgsValidationJob)
+
+
+class TranslatorToVoiceArtistOneOffJobTests(test_utils.GenericTestBase):
```

Comment:
Good tests! Please include one for multiple users being moved over, too.

Reply:
Done.